doc/dox_comments/header_files/aes.h

```diff
@@ -870,3 +871,73 @@ WOLFSSL_API int wc_AesXtsFree(XtsAes* aes);
 */
 WOLFSSL_API int  wc_AesInit(Aes*, void*, int);
 
+/*!
+    \ingroup AES
+
+    \brief AES with CFB mode.
+
+    \return 0 Success and negative error values on failure
+
+    \param aes   AES keys to use for block encrypt/decrypt
+    \param out   output buffer to hold cipher text must be at least as large
+    as inputbuffer)
+    \param in    input plain text buffer to encrypt
+    \param sz    size of input buffer
+
+    _Example_
+    \code
+    Aes aes;
+    unsigned char plain[SIZE];
+    unsigned char cipher[SIZE];
+
+    //set up key with AES_ENCRYPTION as dir for both encrypt and decrypt
+
+    if(wc_AesCfbEncrypt(&aes, cipher, plain, SIZE) != 0)
+    {
+        // Handle error
+    }
+    \endcode
+
+    \sa wc_AesCfbDecrypt
+    \sa wc_AesSetKey
+*/
+int wc_AesCfbEncrypt(Aes* aes, byte* out, const byte* in, word32 sz)
```

Comment:
With this being a header file please adjust the function declarations to be WOLFSSL_API <function> for wc_AesCfbEncrypt and wcAesCfbDecrypt.